docs/buildah-run.md

```diff
@@ -26,7 +26,9 @@ The *path* to an alternate OCI-compatible runtime.
 
 Adds global flags for the container runtime. To list the supported flags, please
 consult manpages of your selected container runtime (`runc` is the default
-runtime, the manpage to consult is `runc(8)`)
+runtime, the manpage to consult is `runc(8)`).
+Note: To pass the runc flag `log-format json` to buildah run, the option given
```

Comment:
Let me see if I can get on @umohnani8 PITA list.  Can you add the '--' before the first log-format?

Note: To pass the runc flag `--log-format json` to buildah run, the option given

(Tom now dives for cover).

Reply:
Haha no, my fault should have thought about that. Fixed now.